### PR TITLE
Updated gradle cache example

### PR DESCRIPTION
**Pull request by odin-delrio**
### Why:
The example using gradle caches is outdated according to https://github.com/actions/cache/blob/main/examples.md#java---gradle (wich I changed with a [PR](https://github.com/actions/cache/pull/507) recently), if copied&pasted could cause gradle builds in GH Actions to not use the cache properly (causing slower builds than needed). 

### What's being changed:
The example using gradle caches.

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by MiladHassani73** [comment]
CODE_OF_CONDUCT.md

**Comment by janiceilene** [comment]
@odin-delrio Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Review by waleed535 — APPROVED** [review]

File: content/actions/guides/building-and-testing-java-with-gradle.md [hunk 1]
```diff
@@ -105,15 +105,18 @@ steps:
...
+      key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+      restore-keys: |
+        ${{ runner.os }}-gradle-
   - name: Build with Gradle
     run: ./gradlew build
 ```
 {% endraw %}
 
-This workflow will save the contents of your local Gradle package cache, located in the `.gradle/caches` directory of the runner's home directory. The cache key will be the hashed contents of the gradle build files, so changes to them will invalidate the cache.
+This workflow will save the contents of your local Gradle package cache, located in the `.gradle/caches` and `.gradle/wrapper` directories of the runner's home directory. The cache key will be the hashed contents of the gradle build files (including the gradle wrapper properties file), so changes to any of them will invalidate the cache.
```
Comment (lucascosti) [inline content/actions/guides/building-and-testing-java-with-gradle.md:119]:
```suggestion
This workflow will save the contents of your local Gradle package cache, located in the `.gradle/caches` and `.gradle/wrapper` directories of the runner's home directory. The cache key will be the hashed contents of the gradle build files (including the Gradle wrapper properties file), so any changes to them will invalidate the cache.
```

**Review by lucascosti — APPROVED** [review]
Thanks a lot for updating this, @odin-delrio! I'll just do a tiny update to the wording, and then merge this in 👍

**Pull request merged**